views/about.ejs

```diff
@@ -8,7 +8,6 @@
 <body>
 
 <% include template/nav.ejs %>
-
```

Comment:
Discard this change since it's not necessary, we would like to keep all PR code changes meaningful